.github/workflows/theseus-engine.yml

```
@@ -6,39 +6,72 @@ on:
     runs-on: ubuntu-latest
-    permissions:
-      contents: write
-      pull-requests: write
+    outputs:
+      repos: ${{ steps.extract.outputs.repos }}
+    steps:
+      - uses: actions/checkout@v4
+      - id: extract
+        run: |
+          REPOS=$(python << 'EOF'
+import json
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)